### DISCUSSION TITLE
About Adding back to the team

### DISCUSSION THREAD

**Discussion by sajidshahriar72543** [ROM-builders/temporary#8576] [General]
Hello @Apon77 , I deleted my old TG account and created this one after 2 months. I was in the Push users group. Can you please add me back? My git: github.com/sajidshahriar72543
Thanks.

**Comment by Apon77** [comment]
> Hello @Apon77 , I deleted my old TG account and created this one after 2 months. I was in the Push users group. Can you please add me back? My git: github.com/sajidshahriar72543 Thanks.

Your github user is already push user, for telegram group I need your teleram username!

**Reply by sajidshahriar72543** [reply]
tg username: @paperboy22

Thanks
